pkger/parser.go

```diff
@@ -155,7 +162,7 @@ func parseSource(r io.Reader, opts ...ValidateOptFn) (*Pkg, error) {
 		strings.Contains(contentType, "yml"):
 		return parseYAML(bytes.NewReader(b), opts...)
 	default:
-		return parseJsonnet(r, opts...)
+		return parseJsonnet(bytes.NewReader(b), opts...)
```

Comment:
same question above

Reply:
this was actually a bug. I drain the reader above to get that `[]byte`, then here I was reading from the drained reader 🤦‍♂.  This fixes that so it creates a new reader that can be read from again. If I had a read seeker here, I could just seek back to the beginining, but network streams being waht they are.... that ain't happening anytime soon lol

Reply:
After a little perusing of the k8s API conventions.... I think I'll rename `Kind` to `Object` which is how k8s represents an entity of a kind. And then switch back `KindType` to `Kind`. 

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#types-kinds